src/main/windows/app.ts

```diff
@@ -129,6 +128,8 @@ export class AppWindow extends BrowserWindow {
     this.on('unmaximize', resize);
 
     this.on('close', (event: Electron.Event) => {
+      const { object: settings } = this.windowsManager.settings;
```

Comment:
Are you sure this is correct?

Reply:
Yes, without it, the settings are static. I tested and works as it should

Reply:
Hold up, I didn't know about this syntax